macOS/DuckDuckGo/LoginItems/LoginItemsManager.swift

```diff
@@ -90,6 +91,12 @@ final class LoginItemsManager: LoginItemsManaging {
         })
     }
 
+    func isAnyInstalled(_ items: Set<LoginItem>) -> Bool {
+        return items.contains(where: { item in
+            item.status.isInstalled
```

Comment:
You  can check is installed for the exact definition.  It means the login item is either enabled or requires user authorization.